### PR TITLE
Add hints for command fe/f

### PR DESCRIPTION
**Pull request by PeiweiHu**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Fix the description of command `fc`
- Add hints for command `fe`
- Expose the hint while using command `f` if a flag already exists. 

**Test plan**

CI is green. 

**Closing issues**

...

### CODE REVIEW

File: librz/core/cmd/cmd_flag.c [hunk 1]
```diff
@@ -776,8 +776,10 @@ RZ_IPI RzCmdStatus rz_flag_base_handler(RzCore *core, int argc, const char **arg
 RZ_IPI RzCmdStatus rz_flag_exists_handler(RzCore *core, int argc, const char **argv) {
 	RzFlagItem *item = rz_flag_get(core->flags, argv[1]);
 	if (!item) {
+		rz_cons_printf("Cannot find flag '%s'\n", argv[1]);
```
Comment (XVilka) [inline librz/core/cmd/cmd_flag.c]: `RZ_LOG_ERROR()`

File: librz/core/cmd/cmd_flag.c [hunk 2]
```diff
@@ -776,8 +776,10 @@ RZ_IPI RzCmdStatus rz_flag_base_handler(RzCore *core, int argc, const char **arg
...
 		return RZ_CMD_STATUS_ERROR;
 	}
+	rz_cons_printf("Find flag '%s' at 0x%" PFMT64x "\n", argv[1], item->offset);
```
Comment (XVilka) [inline librz/core/cmd/cmd_flag.c]: `RZ_LOG_DEBUG()`

**Review by XVilka — CHANGES_REQUESTED** [review]
Looks good apart from those two messages

**Review by ret2libc — APPROVED** [review]
LGTM apart from what said by @XVilka

**Review by XVilka — APPROVED** [review]

**Pull request merged**